common/djangoapps/django_comment_common/migrations/0007_move_id_mapping_to_new_model.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-11 14:21
+from __future__ import unicode_literals
+
+from django.db import migrations
+import jsonfield.fields
+import opaque_keys.edx.django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('django_comment_common', '0006_coursediscussionsettings_discussions_id_map'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DiscussionsIdMapping',
+            fields=[
+                ('course_id', opaque_keys.edx.django.models.CourseKeyField(db_index=True, max_length=255, primary_key=True, serialize=False)),
+                ('mapping', jsonfield.fields.JSONField(help_text=b'Key/value store mapping discussion IDs to discussion XBlock usage keys.')),
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='coursediscussionsettings',
+            name='discussions_id_map',
+        ),
```

Comment:
I think it's safe to drop this column in the same migration because we're not actually writing to it yet.

Reply:
Hmm, won't we get errors if we try to save a CourseDiscussionSettings model between when this migration is run and the new code is fully deployed?

Reply:
Oh, does the ORM generate updates for attributes that haven't changed? I guess it's safer to just split it out then. I'll rearrange the migration and make a separate PR with the drop column migration.

Reply:
I'm not 100% on that, but my gut tells me it could be a problem. We could test it on a devstack or just split it out. Not sure which is quicker in the long run.

Reply:
The drop column migration is no longer a part of this PR.